projectfiles/CodeBlocks/README.md

```diff
@@ -37,6 +35,8 @@
 	
 	C:\masm32\bineditbin.exe /LARGEADDRESSAWARE cc1plus.exe
 	```
+	
+	if you use 64-bit version, then you cannot use MASM32, who don't have 64 bit version, and you cannot make debug build.
```

Comment:
"if you use 64-bit version, then you cannot use MASM32, who don't have 64 bit version, and you cannot make debug build."
should be
"If you use the 64-bit version then you cannot use MASM32 since it doesn't have a 64 bit version, which means you cannot make a debug build."